src/modes/play/objects/player.js

```diff
@@ -11,17 +11,99 @@ class Player extends Mob {
     this.fallSpeed = 4;
     this.accel = 1.5;
     this.frictionCoeff = 0.6;
-    this.spriteBank = {
-      standing: ['./assets/sprites/player/moira/bunny2_stand.png'],
-      running: ['./assets/sprites/player/moira/bunny2_walk1.png', './assets/sprites/player/moira/bunny2_walk2.png'],
-      jumping: ['./assets/sprites/player/moira/bunny2_jump.png'],
+    this.configPower('small');
+  }
+
+  configPower(power) {
+    this.spriteBank = this.spriteSet(power);
+    if (power === 'small') {
+      this.size = { x: 18, y: 18 };
+      this.jumpSpeed = 24;
+    } else if (power === 'large') {
+      this.y -= 14;
+      this.size = { x: 32, y: 32 };
+      this.jumpSpeed = 32;
+    } else {
+      // If power does not match any prescribed statuses, return without
+      // setting new power.
+      return
+    }
+
+    return this.status.power = power;
+  }
+
+  spriteSet(setName) {
+    if (setName === 'small') {
+      return {
+        standing: ['./assets/sprites/player/moira/small/bunny2_stand.png'],
+        running: ['./assets/sprites/player/moira/small/bunny2_walk1.png', './assets/sprites/player/moira/small/bunny2_walk2.png'],
+        jumping: ['./assets/sprites/player/moira/small/bunny2_jump.png'],
+      }
+    } else if (setName === 'large') {
+      return {
+        standing: [
+          './assets/sprites/player/moira/large/idle1.png',
+          './assets/sprites/player/moira/large/idle2.png',
+          './assets/sprites/player/moira/large/idle3.png'
+        ],
+        running: [
+          './assets/sprites/player/moira/large/run1.png',
+          './assets/sprites/player/moira/large/run2.png',
+          './assets/sprites/player/moira/large/run3.png',
+          './assets/sprites/player/moira/large/run4.png',
+          './assets/sprites/player/moira/large/run5.png',
+        ],
+        jumping: ['./assets/sprites/player/moira/large/run2.png'],
+      }
+    }
+  }
+
+  get hitBox() {
```

Comment:
With the larger sprite, you notice some reeeally bonkers stuff with collisions: Basically, you're wide enough to fall through (or rather, fall... around...?) entire blocks. I tried manually interpolating midpoints, and it solved that problem.

It introduced some other new and exciting problems-- like the big sprite can get stuck in ledges fairly effectively-- but I think the "interpolate extra points" approach seems promising.